logging/about-logging.md

```diff
@@ -14,19 +14,19 @@ The description section gives the reader all the key information they need to un
 
 You complete the sections as they relate to your logs and logging pipelines.
 
-You don't need to describe the mandatory logging parameters sent by default to the log pipeline. These parameters should be defined in higher-level reference documentation for your project.
+You don't need to describe the mandatory logging parameters sent by default to the log pipeline. Define these paramaters in the higher-level reference documentation of your project.
 
 Only describe the parameters unique to your application or common parameters that are being reused from other applications.
 
 ### About the "Logging example" Section
 
-If you get called late at night and you're trying to debug a problem, not knowing how to interpret a logfile because there is no example of what to expect is extremely frustrating.
+It can be extremely frustrating to get a call late at night when you're trying to debug a problem, if you don't provide an example of how to interpret a logfile.
```

Comment:
My initial wording of this was awkward and this hasn't helped you understand what I was suggesting in this sentence.

Maybe something like:

"Not having good log file examples can make it frustrating to debug a problem when you are on-call."

Reply:
@jaredmorgs I made the change. The next sentence too sounds a bit odd. I changed that as well.

Reply:
Yep that did need some tweaking as well. Thanks for the fix.